user-guide/Getting_started/DaaS/Order_DataMiner_credits.md

```diff
@@ -0,0 +1,48 @@
+---
+uid: Order_DataMiner_credits
+---
+
+# Ordering DataMiner credits
+
+To create and maintain a DaaS instance, your [organization](xref:Pricing_Usage_based_service#organization) must have sufficient DataMiner credits. To order DataMiner credits for the first time, follow the procedure below.
+
+> [!NOTE]
+> If you have already created a SaaS subscription as detailed below, and you want to order **additional credits**, you do not have to create a new SaaS subscription. Instead, go to your SaaS subscription in the Azure portal, and click *Open SaaS Account on publisher's site* to be redirected to the *Purchase credits* page.
+
+1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/) and search for DataMiner.
+
+   ![Azure Marketplace DataMiner credits offer](~/user-guide/images/Azure_Marketplace_Offer.png)
+
+1. Click *Get It Now* and continue to the Azure portal.
+
+   > [!IMPORTANT]
+   > Make sure you have selected the plan DataMiner credits.
+
+1. Once you have been redirected to the Azure portal, click *Subscribe*.
+
+   This will start the process of creating a SaaS subscription resource to be deployed in a resource group.
+
+1. Using your new SaaS subscription, click *Configure account now*.
+
+   This will redirect you to our marketplace landing page.
+
+   ![Configure account now](~/user-guide/images/Azure_Portal_Configure.png)
+
+1. After confirming all you details, click *Purchase credits*.
```

Comment:
What do you mean with "confirming all your details"? If this is a separate action the user needs to do, it would be better to make this a separate step before this one.

Reply:
the initial screen shows the following:
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/146821343/fcf02b3b-eaa8-4943-afae-391de98e5854)
with this step i mean that the user should just look over their personal details (email) and if they are using the correct account they can click *Purchase credits*